### PR TITLE
i18n(es): translate markdown-content

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [x] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

- translates `guides/markdown-content.md` to Spanish
- updates `guides/markdown-content.md` to match #668

### CODE REVIEW

File: src/pages/es/guides/markdown-content.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+### Resaltado de sintaxis
+
+Astro viene con soporte integrado para [Shiki](https://shiki.matsu.io/) y [Prism](https://prismjs.com/). Esto proporciona un resaltado de sintaxis instantáneo para:
+
+- todas las vallas de codigo (\`\`\`) usadas en los archivos markdown (`.md`) y el [componente `<Markdown />`](#componente-markdown).
+- el contenido dentro del [componente `<Code />`](/es/reference/api-reference/#code-) (con la tecnología de Shiki), o el [componente `<Prism />`](/es/reference/api-reference/#prism-) impulsado por Prism.
+
+Shiki está habilitado de forma predeterminada, preconfigurado con el tema `github-dark`. La salida compilada se limitará a estilos en línea sin clases CSS extrañas, hojas de estilo o JS del lado del cliente.
+
+Si opta por usar Prism, aplicaremos las clases CSS de Prism en su lugar. ¡Tenga en cuenta que **necesita traer su propia hoja de estilo CSS** para que aparezca el resaltado de sintaxis! Consulte la [sección de configuración del Prism](#configuración-de-prism) para obtener más detalles.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:391]:
```suggestion
Si opta por usar Prism, aplicaremos las clases CSS de Prism en su lugar. ¡Tenga en cuenta que **necesita traer su propia hoja de estilo CSS** para que aparezca el resaltado de sintaxis! Consulte la [sección de configuración de Prism](#configuración-de-prism) para obtener más detalles.
```

File: src/pages/es/guides/markdown-content.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+      // https://github.com/shikijs/shiki/blob/main/docs/languages.md
+      langs: [],
+      // Habilite word wrap para evitar el desplazamiento horizontal
+      wrap: true,
+    },
+  },
+};
+```
+
+También sugerimos [leer la documentación de su tema](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) para explorar la carga de un tema personalizado, alternar entre el modo claro y el oscuro, o estilar a través de variables de CSS.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:430]:
```suggestion
También sugerimos [leer la documentación de Shiki sobre sus temas](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) para explorar la carga de un tema personalizado, alternar entre el modo claro y el oscuro, o estilar a través de variables de CSS.
```

File: src/pages/es/guides/markdown-content.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+
+#### Configuración de Prism
+
+Cuando use Prism, deberá agregar una hoja de estilos a su proyecto para resaltar la sintaxis. Si recién empieza y prefiere usar Prism en lugar de Shiki, le sugerimos:
+<!-- todo -->
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:435]: I think you left this here by mistake

File: src/pages/es/guides/markdown-content.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Markdown
+description: Usando Markdown en Astro
+i18nReady: true
+---
+
+El contenido de markdown se usa comúnmente para crear contenido con mucho texto, como artículos de blog y documentación. Astro incluye soporte integrado para markdown con algunas características adicionales como soporte para expresiones JavaScript y componentes de Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:8]:
```suggestion
El contenido de Markdown se usa comúnmente para crear contenido con mucho texto, como artículos de blog y documentación. Astro incluye soporte integrado para Markdown con algunas características adicionales como soporte para expresiones JavaScript y componentes de Astro.
```

File: src/pages/es/guides/markdown-content.md [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+
+## Páginas de markdown
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:10]:
```suggestion
## Páginas de Markdown
```

File: src/pages/es/guides/markdown-content.md [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+
+Astro trata cualquier archivo `.md` dentro de la carpeta `/src/pages` como una página. Al colocar un archivo en esta carpeta, o en cualquier subcarpeta, se creará automáticamente una ruta de página utilizando la ruta del archivo.
+
+📚 Obtenga más información sobre [enrutamiento basado en archivos](/es/core-concepts/routing/) en Astro.
+
+### Ejemplo básico
+
+La forma más fácil de comenzar a usar markdown en Astro es crear una ruta de la página de inicio `src/pages/index.md` en su proyecto. Copie el codigo básico a continuación en su proyecto y luego vea el HTML renderizado en la página de inicio de su proyecto. Por lo general, se encuentra en [http://localhost:3000/](http://localhost:3000/).
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:18]:
```suggestion
La forma más fácil de comenzar a usar Markdown en Astro es crear una ruta de la página de inicio `src/pages/index.md` en su proyecto. Copie el codigo a continuación en su proyecto y luego podrá ver el HTML renderizado en la página de inicio de su proyecto. Por lo general, se encuentra en [http://localhost:3000/](http://localhost:3000/).
```

File: src/pages/es/guides/markdown-content.md [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+Esta es su primera página de markdown. Probablemente no tenga mucho estilo, aunque
+markdown soporta **negrita** y _cursiva._
+
+Para obtener más información sobre cómo agregar una plantilla a su página, 
+lea la siguiente sección sobre **Plantillas de Markdown**
+```
+
+### Plantillas de markdown
+
+Las páginas de markdown tienen una propiedad frontmatter especial para `layout` que define **la ruta relativa** a un [componente plantilla](/es/core-concepts/layouts/) de Astro. Este componente envolverá su contenido markdown, proporcionando una capa común que incluirá cualquier otro elemento plantilla que desee agregar.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:37]:
```suggestion
Las páginas de Markdown tienen una propiedad frontmatter especial para `layout` que define **la ruta relativa** a un [componente plantilla](/es/core-concepts/layouts/) de Astro. Este componente envolverá su contenido Markdown, proporcionando una capa común que incluirá cualquier otro elemento plantilla que desee agregar.
```

File: src/pages/es/guides/markdown-content.md [hunk 8]
```diff
@@ -0,0 +1,441 @@
...
+
+```markdown
+---
+# Example: src/pages/index.md
+title: Hola mundo
+---
+
+# Hola!
+
+Esta es su primera página de markdown. Probablemente no tenga mucho estilo, aunque
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:28]:
```suggestion
Esta es su primera página de Markdown. Probablemente no tenga mucho estilo, aunque
```

File: src/pages/es/guides/markdown-content.md [hunk 9]
```diff
@@ -0,0 +1,441 @@
...
+markdown soporta **negrita** y _cursiva._
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:29]:
```suggestion
Markdown soporta **negrita** y _cursiva._
```

File: src/pages/es/guides/markdown-content.md [hunk 10]
```diff
@@ -0,0 +1,441 @@
...
+
+Las páginas de markdown tienen una propiedad frontmatter especial para `layout` que define **la ruta relativa** a un [componente plantilla](/es/core-concepts/layouts/) de Astro. Este componente envolverá su contenido markdown, proporcionando una capa común que incluirá cualquier otro elemento plantilla que desee agregar.
+
+```markdown
+---
+layout: ../layouts/BaseLayout.astro
+---
+```
+
+Un diseño típico de las páginas de markdown incluyen:
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:45]:
```suggestion
Un diseño típico de las páginas de Markdown incluyen:
```

File: src/pages/es/guides/markdown-content.md [hunk 11]
```diff
@@ -0,0 +1,441 @@
...
+
+1. la propiedad `content` para acceder a los metadatos de la página de markdown.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:47]:
```suggestion
1. la propiedad `content` para acceder a los metadatos de la página de Markdown.
```

File: src/pages/es/guides/markdown-content.md [hunk 12]
```diff
@@ -0,0 +1,441 @@
...
+2. un [`<slot />`](/es/core-concepts/astro-components/#slots) predeterminado para indicar dónde debe mostrarse el contenido markdown de la página.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:48]:
```suggestion
2. un [`<slot />`](/es/core-concepts/astro-components/#slots) predeterminado para indicar dónde debe mostrarse el contenido Markdown de la página.
```

File: src/pages/es/guides/markdown-content.md [hunk 13]
```diff
@@ -0,0 +1,441 @@
...
+  },
+  "url": ""
+}
+```
+
+> 💡 `astro` y `url` son las únicas propiedades garantizadas proporcionadas por Astro en la propiedad `content`. El resto del objeto está definido por sus variables de frontmatter.
+
+### Frontmatter como props
+
+Cualquier componente de Astro (¡no solo las plantillas!) pueden recibir valores definidos en el frontmatter del markdown como props. También puedes especificar varios tipos de datos usando YAML en el frontmatter y capturar metadatos de cada publicación de blog para usar en su proyecto de Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:109]:
```suggestion
Cualquier componente de Astro (¡no solo las plantillas!) pueden recibir valores definidos en el frontmatter del Markdown como props. También puedes especificar varios tipos de datos usando YAML en el frontmatter y capturar metadatos de cada publicación de blog para usar en su proyecto de Astro.
```

File: src/pages/es/guides/markdown-content.md [hunk 14]
```diff
@@ -0,0 +1,441 @@
...
+
+Acceda a estos valores en cualquier archivo `.astro` como lo haría en un componente plantilla, como se describe anteriormente.
+
+### IDs de título
+
+Astro agregará identificaciones generadas automáticamente a todos los títulos y subtítulos en los archivos markdown usando [github-slugger](https://github.com/Flet/github-slugger). Pero, si se especifica una identificación personalizada, no será sobreescrita.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:115]:
```suggestion
Astro agregará identificaciones generadas automáticamente a todos los títulos y subtítulos en los archivos Markdown usando [github-slugger](https://github.com/Flet/github-slugger). Pero, si se especifica una identificación personalizada, no será sobreescrita.
```

File: src/pages/es/guides/markdown-content.md [hunk 15]
```diff
@@ -0,0 +1,441 @@
...
+
+Estas identificaciones se agregarán _después_ de que se ejecuten todos los demás complementos, por lo que si tiene un complemento como `rehype-toc` que necesita identificaciones, deberá agregar su propio plugin de slug (como `rehype-slug`).
+
+### Borradores en markdown
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:119]:
```suggestion
### Borradores en Markdown
```

File: src/pages/es/guides/markdown-content.md [hunk 16]
```diff
@@ -0,0 +1,441 @@
...
+
+`draft: true` es un valor opcional que marcará una página o artículo individual `.md` como "borrador". De forma predeterminada, esta página se excluirá de la compilación final de su proyecto.
+
+Las páginas de markdown sin la propiedad `draft` o aquellas con `draft: false` no se verán afectadas y se incluirán en la compilación final.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:123]:
```suggestion
Las páginas de Markdown sin la propiedad `draft` o aquellas con `draft: false` no se verán afectadas y se incluirán en la compilación final.
```

File: src/pages/es/guides/markdown-content.md [hunk 17]
```diff
@@ -0,0 +1,441 @@
...
+
+No se creará ninguna página para esta publicación.
+
+Para crear y publicar esta publicación:
+
+- actualice el frontmatter a `draft: false` o
+- elimina la propiedad `draft` por completo.
+```
+
+> ⚠️ Aunque `draft: true` evitará que se construya la página de su proyecto, este archivo estará disponible para `Astro.glob()` el cual devuelve **todos los archivos markdown** en la ruta especificada.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:143]:
```suggestion
> ⚠️ Aunque `draft: true` evitará que se construya la página de su proyecto, este archivo estará disponible para `Astro.glob()` el cual devuelve **todos los archivos Markdown** en la ruta especificada.
```

File: src/pages/es/guides/markdown-content.md [hunk 18]
```diff
@@ -0,0 +1,441 @@
...
+export default defineConfig({
+  markdown: {
+    drafts: true,
+  },
+});
+```
+
+💡 ¡También puedes agregar la extensión `--drafts` al ejecutar `astro build` para incluir la creación de páginas borrador!
+
+## Autoría de markdown
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:162]:
```suggestion
## Autoría de Markdown
```

File: src/pages/es/guides/markdown-content.md [hunk 19]
```diff
@@ -0,0 +1,441 @@
...
+
+Además de admitir la sintaxis estándar de markdown, Astro también amplía markdown para que su contenido sea aún más expresivo. A continuación se muestran algunas características de markdown que solo existen en Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:164]:
```suggestion
Además de admitir la sintaxis estándar de Markdown, Astro también extiende Markdown para que su contenido sea aún más expresivo. A continuación se muestran algunas características de markdown que solo existen en Astro.
```

File: src/pages/es/guides/markdown-content.md [hunk 20]
```diff
@@ -0,0 +1,441 @@
...
+
+### Usando de variables en markdown
+
+Las variables de frontmatter se pueden usar directamente en su markdown como propiedades del objeto `frontmatter`.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:168]:
```suggestion
Las variables de frontmatter se pueden usar directamente en su Markdown como propiedades del objeto `frontmatter`.
```

File: src/pages/es/guides/markdown-content.md [hunk 21]
```diff
@@ -0,0 +1,441 @@
...
+author: Leon
+age: 42
+---
+
+# Acerca del author
+
+{frontmatter.author} tiene {frontmatter.age} y vive en Toronto, Canada.
+```
+
+### Usando componentes en markdown
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:181]:
```suggestion
### Usando componentes en Markdown
```

File: src/pages/es/guides/markdown-content.md [hunk 22]
```diff
@@ -0,0 +1,441 @@
...
+author: Leon
+---
+
+<Author name={frontmatter.author}/>
+<Biography client:visible>
+  {frontmatter.author} vive en Toronto, Cánada y disfruta de la fotografía.
+</Biography>
+```
+
+## Importando markdown
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:200]:
```suggestion
## Importando Markdown
```

File: src/pages/es/guides/markdown-content.md [hunk 23]
```diff
@@ -0,0 +1,441 @@
...
+---
+
+Genial artículo: <a href={greatPost.url}>{greatPost.frontmatter.title}</a>
+
+<ul>
+  {posts.map(post => <li>{post.frontmatter.title}</li>)}
+</ul>
+```
+
+Cada archivo markdown exporta las siguientes propiedades:
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:220]:
```suggestion
Cada archivo Markdown exporta las siguientes propiedades:
```

File: src/pages/es/guides/markdown-content.md [hunk 24]
```diff
@@ -0,0 +1,441 @@
...
+
+- `frontmatter`: cualquier dato especificado en el frontmatter YAML de este archivo.
+- `file`: La ruta absoluta de este archivo (por ejemplo, `/home/user/projects/.../file.md`).
+- `url`: si es una página, URL de la página (por ejemplo, `/es/guides/markdown-content`).
+- `getHeaders()`: una función asíncrona que devuelve los títulos del archivo markdown. La respuesta sigue este tipo: `{ depth: number; slug: string; text: string }[]`.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:225]:
```suggestion
- `getHeaders()`: una función asíncrona que devuelve los títulos del archivo Markdown. La respuesta sigue este tipo: `{ depth: number; slug: string; text: string }[]`.
```

File: src/pages/es/guides/markdown-content.md [hunk 25]
```diff
@@ -0,0 +1,441 @@
...
+- `Content`: un componente que representa el contenido del archivo markdown. Aquí hay un ejemplo:
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:226]:
```suggestion
- `Content`: un componente que representa el contenido del archivo Markdown. Aquí hay un ejemplo:
```

File: src/pages/es/guides/markdown-content.md [hunk 26]
```diff
@@ -0,0 +1,441 @@
...
+  {posts.map(post => <li>{post.title}</li>)}
+  <!-- post.title será un `string`! -->
+</ul>
+```
+
+## Componente Markdown
+
+> NOTA: El componente `<Markdown />` no funciona en SSR y puede eliminarse antes de v1.0. Debe evitarse si es posible. Para usar Markdown en sus plantillas, use un archivo `.md` separado y luego [`import` Markdown](/es/guides/markdown-content/#importando-markdown) en su plantilla como componente.
+
+Puede importar el [componente markdown](/es/reference/api-reference/#markdown-) de Astro en el script de su componente y escribir cualquier markdown que desee entre las etiquetas `<Markdown></Markdown>`.
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:258]:
```suggestion
Puede importar el [componente Markdown](/es/reference/api-reference/#markdown-) de Astro en el script de su componente y escribir cualquier markdown que desee entre las etiquetas `<Markdown></Markdown>`.
```

File: src/pages/es/guides/markdown-content.md [hunk 27]
```diff
@@ -0,0 +1,441 @@
...
+    - Normalización automática de indentación
+    - Escape automático de expresiones dentro de bloques de código.
+
+    ```js
+      // ¡Este contenido no se transformará!
+      const object = { someOtherValue };
+    ```
+
+    - Compatibilidad con componentes de UI como cualquier archivo `.astro`.
+    - Compatibilidad con markdown recursivo (los componentes secundarios también se procesan como Markdown)
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:286]:
```suggestion
    - Compatibilidad con Markdown recursivo (los componentes secundarios también se procesan como Markdown)
```

File: src/pages/es/guides/markdown-content.md [hunk 28]
```diff
@@ -0,0 +1,441 @@
...
+```astro
+---
+import { Markdown } from 'astro/components';
+
+const content = await fetch('https://raw.githubusercontent.com/withastro/docs/main/README.md').then(res => res.text());
+---
+
+<Layout>
+  <Markdown>
+    ## Ejemplo markdown
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:323]:
```suggestion
    ## Ejemplo Markdown 
```

**Review by agustinmulet — COMMENTED** [review]
Some suggestions, I liked Markdown with capital M better 😝

**Comment by sarah11918** [comment]
Just let me know when this one is ready, @kevinzunigacuellar !

File: src/pages/es/guides/markdown-content.md [hunk 29]
```diff
@@ -0,0 +1,476 @@
...
+  {posts.map((post) => (
+    <Fragment>
+      <h2>{post.frontmatter.title}</h2>
+      <p>{readingTime(post.rawContent()).text}</p>
+    </Fragment>
+  ))}
+  ```
+
+
+- `compiledContent()`: Una función asíncrona que devuelve el contenido compilado a una sintaxis de Astro válida. Nota: **¡Esto no analiza `{expresiones jsx}`, `<Componentes />` o componentes plantilla**! Solo los bloques de Markdown estándar como `##títulos` y `-listas` se compilaran a HTML. Esto es útil cuando, por ejemplo, tienes un bloque de resumen dentro del artículo de blog. Dado que la sintaxis de Astro es HTML válido, podemos usar bibliotecas populares como [node-html-parser](https://www.npmjs.com/package/node-html-parser) para consultar el primer párrafo de la siguiente manera:
```
Comment (agustinmulet) [inline src/pages/es/guides/markdown-content.md:242]:
```suggestion
- `compiledContent()`: Una función asíncrona que devuelve el contenido compilado a una sintaxis de Astro válida. Nota: **¡Esto no analiza `{expresiones jsx}`, `<Componentes />` o componentes plantilla**! Solo los bloques de Markdown estándar como `##títulos` y `-listas` se compilarán a HTML. Esto es útil cuando, por ejemplo, tienes un bloque de resumen dentro del artículo de blog. Dado que la sintaxis de Astro es HTML válido, podemos usar bibliotecas populares como [node-html-parser](https://www.npmjs.com/package/node-html-parser) para consultar el primer párrafo de la siguiente manera:
```

**Review by agustinmulet — COMMENTED** [review]
One simple change

**Comment by sarah11918** [comment]
(Hope it's OK! I just merged the quick change while I was in here, and updated the branch so it will be good to merge as soon as you say so!)

**Comment by kevinzunigacuellar** [comment]
@sarah11918 the page is good to go 🚀

**Pull request merged**